lib/railway_ipc/consumer/consumer.rb

```diff
@@ -90,14 +91,14 @@ def process_unknown_message_type
 
     def create_message_with_status!(status)
       RailwayIpc::ConsumedMessage.create!(
-        uuid: protobuff_message.uuid,
+        uuid: protobuf_message.uuid,
         status: status,
         message_type: decoded_payload.type,
-        user_uuid: protobuff_message.user_uuid,
-        correlation_id: protobuff_message.correlation_id,
+        user_uuid: protobuf_message.user_uuid,
+        correlation_id: protobuf_message.correlation_id,
         queue: delivery_info.consumer.queue.name,
         exchange: delivery_info.exchange,
-        encoded_message: decoded_payload.message
+        encoded_message: encoded_message
```

Comment:
Previously tried to store a binary. Storing raw payload like we do with publishing instead.